plugin/src/main/java/com/playmonumenta/scriptedquests/timers/CommandTimer.java

```diff
@@ -157,6 +182,17 @@ private boolean tryAddTimer(final ArmorStand entity, final Set<String> tags, fin
 		return true;
 	}
 
+	//Put the new timer in a bucket
+	//This implementation can be improved, right now it just puts it in the one after the last one placed
+	private void scheduleTimer(UUID uniqueId, CommandTimerInstance timer) {
+		mTickTimers.get(mScheduler).put(uniqueId, timer);
+		mScheduler++;
+		if (mScheduler >= mPeriod) {
+			mScheduler = 0;
+		}
```

Comment:
This will still likely end up with some things in the same bucket and some buckets empty as things load and unload over time. 

Can we just loop over each thing, and check size(). Maybe a variable to keep track of the index of the lowest seen size, and stop immediately if it finds a 0.